Eatery/Controllers/MapViewController.swift

```diff
@@ -96,30 +105,37 @@ class MapViewController: UIViewController, MKMapViewDelegate, CLLocationManagerD
     {
         // Create bottom left arrow button
         arrowButton.frame = CGRect(x: 15, y: view.frame.size.height - 55, width: 30, height: 30)
-        arrowButton.setImage(UIImage(named: "locationArrowIcon"), for: UIControlState())
+        arrowButton.setImage(#imageLiteral(resourceName: "locationArrowIcon"), for: UIControlState())
         arrowButton.addTarget(self, action: #selector(MapViewController.arrowButtonPressed), for: .touchUpInside)
```

Comment:
selector doesn't need the MapViewController.

should just be arrowButtonPressed